### PR TITLE
Clean-up related to: https://jira.duraspace.org/browse/FCREPO-1878

### PR DESCRIPTION
**Pull request by awoods**

### CODE REVIEW

File: fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java [hunk 1]
```diff
@@ -291,7 +291,7 @@ public Response createOrReplaceObjectRdf(
             @HeaderParam("Content-Disposition") final ContentDisposition contentDisposition,
             @HeaderParam("If-Match") final String ifMatch,
             @HeaderParam("Link") final String link)
-            throws IOException, InvalidChecksumException, MalformedRdfException {
+            throws InvalidChecksumException, MalformedRdfException {
```
Comment (acoburn) [inline fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java]: `MalformedRdfException` is a `RuntimeException`, so is it necessary to declare it?

Comment (awoods) [inline fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java]: Probably not necessary.

Comment (acoburn) [inline fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java]: OK, perhaps that can be cleaned up as part of another ticket -- there are plenty of method signatures in the codebase with declared `RuntimeException`s.

Comment (awoods) [inline fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java]: Agreed... lots of opportunities for future cleanup.

**Pull request merged**